tools/status/report_observability.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# This script is only used to provide a temporary but common reporting of connector tests.
```

Comment:
@alafanechere Can we put in a TODO note as to when this should be removed and whos responsible?